guide/14-deep-learning/exploring_model_extensibility_for_named_entity_recognition.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
<p>Consider using <strong>Third Party Language Model </strong>instead of<strong> Model Extensibility </strong>everywhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>This may not require because it is specific to Model extension not to Third Party language model</p>